tests/ppl/explain.yaml

```diff
@@ -0,0 +1,22 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test how a query is executed against OpenSearch.
+
+prologues:
+  - path: /books
+    method: PUT
+    request_body:
+      payload: {}
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Get explain of SQL Query
+    path: /_plugins/_ppl/_explain
+    method: POST
+    request_body:
+      payload:
+        query: search source=books;
```

Comment:
Curious, does it require that semicolon?

Reply:
I just checked, it works without semicolons.
There are no semicolons in the examples of working with the OS API. That's why I removed them in the commit